services/basic_types.proto

```diff
@@ -79,9 +79,30 @@ message AccountID {
     int64 realmNum = 2;
 
     /**
-     * A nonnegative account number unique within its realm
+     * The account number unique within its realm which can be either a non-negative integer or an alias public key.
      */
-    int64 accountNum = 3;
+    oneof account {
+
+        /**
+         * A non-negative account number unique within its realm
+         */
+        int64 accountNum = 3;
+
+        /**
+         * An alias base64 single public key. Only a primitive key is supported as an alias Key (ThresholdKey and KeyList
```

Comment:
Since it's stored as bytes inside the `Key` it can't be base64.
```suggestion
         * A public key to be used as the account's alias. Only a primitive key is supported as an alias Key (ThresholdKey and KeyList
```

Reply:
Fixed. Thanks @steven-sheehy